### PR TITLE
feat: Add acknowledgements to settings

### LINKED ISSUE #355: Add acknowledgments in Settings
Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #355 
videotogif_2018 07 21_12 44 48

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -93,6 +93,8 @@
     <string name="rating_settings">Rate Us</string>
     <string name="app_name_capital">Open Event Android</string>
     <string name="suggestion_settings">Suggest Improvement</string>
+    <string name="key_acknowledgements">acknowledgements</string>
+    <string name="acknowledgements_settings">Acknowledgements</string>
```
[the remark this answers is not in the document]
Comment (dreadpool2) [reply app/src/main/res/values/strings.xml]: We are referring this string in Settings page

[the remark this answers is not in the document]
Comment (dreadpool2) [reply app/src/main/res/values/strings.xml]: Okay got your point, will change that :+1:

[the remark this answers is not in the document]
Comment (dreadpool2) [reply app/src/main/res/values/strings.xml]: Updated

File: app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt [hunk 2]
```diff
@@ -63,6 +64,11 @@ class SettingsFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
             showDialog()
             return true
         }
+        if (preference?.key == resources.getString(R.string.key_acknowledgements)) {
+            //Logout Dialog shown
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: log out dialog?

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/settings/SettingsFragment.kt]: Updated

**Pull request merged**